startleft/cli.py

```diff
@@ -81,21 +78,16 @@ def run(type, map, otm, name, id, recreate, irius_server, api_token, filename):
     """
     Parses IaC source files into Open Threat Model and immediately uploads threat model to IriusRisk
     """
+    logger.info("Parsing IaC source files into OTM")
+    otm_project = OtmProject.from_iac_file(id, name, type, filename, map, otm)
+    otm_service = OtmProjectService(IriusriskProjectRepository(api_token, irius_server))
```

Comment:
From this line till the end of this cli command, the code is exactly the same as in **_threadmodel_** cli command. Could be a good idea to create a common method for that.